kobo-uncaged/main.go

```diff
@@ -591,7 +591,7 @@ func (ku *KoboUncaged) updateNickelDB() error {
 		}
 		_, err = stmt.Exec(desc, series, seriesNum, seriesNumFloat, cid)
 		if err != nil {
-			log.Print(err)
+			log.Print(wrapPos(wrap(err, "could exec nickel db update with %#v", []interface{}{desc, series, seriesNum, seriesNumFloat, cid})))
```

Comment:
`could` -> `could not`?

Reply:
Oops ... I've done that so many times before :smile: